2-mandatory/6-journey-planner.js

```diff
@@ -64,7 +64,10 @@ function checkCodeIsThere(stringText) {
 
   Hint: Use the corresponding array method to split the array.
 */
-function getTransportModes() {}
+function getTransportModes(array) {
+  let transportModesArr = array.slice(1, array.length);
```

Comment:
the default end of the .slice method is the end of the array, so you can omit the `array.length` here :)